avro-kafkaconnect-converter-msk-iam/pom.xml

```diff
@@ -0,0 +1,183 @@
+<!--
```

Comment:
Let's remove this module as discussed. Thanks for incorporating all the feedback!